openforcefield/topology/topology.py

```diff
@@ -2184,6 +2185,7 @@ def to_file(self, filename, positions, file_format="PDB", keepIds=False):
             PDBFile.writeFile(openmm_top, positions, outfile, keepIds)
 
     @staticmethod
+    @requires_package("mdtraj")
```

Comment:
This doesn't actually import MDTraj (someone could make an `mdtraj.Topology`-like object and pass it in here this just fine). So we should remove this.
 
```suggestion
```